src/maps/deploymentStatsFilter.ts

```diff
@@ -11,25 +10,9 @@ export const mapDeploymentStatsFilterToFlowRunsFilter: MapFunction<DeploymentSta
       id: [source.deploymentId],
     },
     flowRuns: {
-      expectedStartTimeAfter: subSeconds(now, source.timeSpanInSeconds),
-      expectedStartTimeBefore: now,
+      startTimeBefore: now,
     },
   }
 
   return filter
 }
-
-export const mapDeploymentStatsFilterToTaskRunsFilter: MapFunction<DeploymentStatsFilter, TaskRunsFilter> = function(source) {
```

Comment:
Checked in this repo and in nebula / oss and found no use. Plus it was named to map to Deployments but was mapping to Flows.. I probably wrote it 😭